### PR TITLE
fix: scale options rangeMin and rangeMax

### PR DESCRIPTION
**Pull request by lvisei**
##### Checklist

- [ ] `npm test` passes
- [ ] benchmarks are included
- [ ] commit message follows commit guidelines
- [ ] documents are updated

##### Description of change

fixed scale options rangeMax and rangeMax

### CODE REVIEW

File: src/runtime/scale.ts [hunk 1]
```diff
@@ -360,13 +360,13 @@ function inferScaleRange(
...
       const [r0, r1] = inferRangeQ(name, colors);
-      return [rangeMin || r0, rangeMax || r1];
+      return [rangeMin ?? r0, rangeMax ?? r1];
     }
     case 'band':
     case 'point': {
       const min = name === 'size' ? 5 : 0;
       const max = name === 'size' ? 10 : 1;
-      return [rangeMin || min, rangeMax || max];
+      return [rangeMin ?? min, rangeMax ?? max];
```
Comment (hustcc) [inline src/runtime/scale.ts:369]: 差别是是否处理了 0。

**Comment by hustcc** [comment]
加个单侧！

File: src/runtime/scale.ts (hunk 1 above)
Comment (lvisei) [reply src/runtime/scale.ts:369]: yep~

**Comment by pearmini** [comment]
同理 https://github.com/antvis/G2/pull/6126#issuecomment-1996280510

**Pull request merged**